smartdevicelink/sdl_ios
# Add support for Jazz Hands

Adds support for [Jazzy](https://github.com/realm/jazzy) to build docs using a custom theme that will later be pulled into the documentation portal.

### Installation

``` bash
[sudo] gem install jazzy
```

This PR is **ready** for review.

### Risk
This PR makes **no** API changes, just adds a build step and requires jazzy to be installed.

### Testing Plan
1. Install Jazzy
2. Build. Ensure Docs are updated to reflect any doc changes.

### Summary
Add a build step for jazzy using a custom theme and include the docs in the repo.
